hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/ScmBlockLocationProtocolClientSideTranslatorPB.java

```diff
@@ -170,6 +171,9 @@ private SCMBlockLocationResponse handleError(SCMBlockLocationResponse resp)
       requestBuilder.setFactor(
           ((RatisReplicationConfig) replicationConfig).getReplicationFactor());
       break;
+    case EC:
+      requestBuilder.setEcReplicationConfig(
+          ((ECReplicationConfig)replicationConfig).toProto());
```

Comment:
Looks like we missed break here. Sorry missed this in previous review.

Reply:
Since this is merged, Let's handle in next patch.

Reply:
Ah. Well spotted. I will create a small patch today to fix this.